Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "mongodm"
-version = "0.9.2"
+version = "1.0.0"
```

suggestion: Since we are still pre 1.0.0, it's fine to bump the minor number only. Cargo is treating those as breaking.

```suggestion
version = "0.10.0"
```

---

changes done by using the minor version as requested